### ISSUE TITLE
[BUG] TerraFinder of Classic Mainnet doesn't show Transactions but TerraStation does

### ISSUE THREAD

**Issue by TheArchitect123** [classic-terra/core#151]
**Describe the bug**
Basically the issue is in the title.

Here's an example transaction I did just now. And I can see it on my TerraStation wallet history, and I can see that the funds between the test wallets reduce or increase accordingly, as they send/receive LUNC.
![image](https://user-images.githubusercontent.com/20945125/221528512-be298b55-e42f-47d0-920f-7fdf9155683b.png)

However when I open up TerraFinder for Classic Mainnet, none of the transactions ever show up. It loops endlessly in a Search Mechanism for the Transaction. I know the Url is correct as well. Here's the same transaction above for TerraFinder.

https://finder.terrac.dev/classic/tx/57247B9E48110AC9159022A6F0EE5E0D5EB7592F35950322D88D4A6D4871D782

This will be a problem for dApps when users try to search for receipts.

**Comment by StrathCole** [comment]
Imho the finder you mentioned is to be used for testnet mainly currently.

The normal finder shows the tx correctly: https://finder.terra.money/classic/tx/57247b9e48110ac9159022a6f0ee5e0d5eb7592f35950322d88d4a6d4871d782

dApps won't use "finder" for these things and as lcd query is working, this is only related to the testnet finder Ed set up I reckon.

**Comment by edk208** [comment]
yes, sorry I took down the fcd and finder for mainnet a few days ago.  My machine was registering 10+ 100% processes on the database, forcing my node to drop blocks... i wanted to keep my node "clear" so I wouldn't run into problems during the upgrade to v1.1.0.  I'll bring them back up after the upgrade

**Comment by ZaradarBH** [comment]
Because that was how the system was built. If you want to change out the storage DB your more then welcome to do that, but no-SQL dbs do not offer better performance in this scenario, so its not really worth the effort.
